source/server/options_impl.cc

```diff
@@ -11,6 +11,10 @@
 #include "spdlog/spdlog.h"
 #include "tclap/CmdLine.h"
 
+#if defined(__APPLE__)
+#define uint64_t unsigned long
```

Comment:
Hrrm, this is unfortunate. I don't understand why the define doesn't cause the same problem.

If we can't figure out a better solution please do add a detailed comment explaining why it's done this way.